packages/rule-tester/src/resolveReportedSuggestions.ts

```diff
@@ -21,19 +22,23 @@ export function resolveReportedSuggestions(
 		return undefined;
 	}
 
-	return suggestionsReported.map((suggestionReported) => ({
-		files: isSuggestionForFiles(suggestionReported)
-			? resolveReportedSuggestionForFiles(
-					suggestionReported,
-					testCaseNormalized,
-				)
-			: [
-					{
-						[testCaseNormalized.fileName]: suggestionReported,
-					},
-				],
```

Comment:
Previously there would _always_ be a `files: { ... }`. The current/single file case would have the current file as the only entry. But I don't want devs to have to put some ad-hoc `file.ts` by default.